src/lib/components/molecules/content-strip/content-strip.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
```

Comment:
is this component necessary? I mean, it's just a wrapper.. for example, in the hero buttons, we're not creating a component to wrap the buttons, so why do it here? :pray:

Reply:
I think I can remove it from the stories and from exporting, but I think it's better to keep it so the hero component doesn't grow too much